src/processors/models/processorsManager.js

```diff
@@ -68,6 +68,23 @@ module.exports.redownloadJSProcessor = async function(processorId) {
     }
 };
 
+module.exports.updateProcessor = async function (processorId, processor) {
+    const oldProcessor = await databaseConnector.getProcessor(processorId);
+    if (!oldProcessor) {
+        throw generateProcessorNotFoundError();
+    }
+    processor.created_at = oldProcessor.created_at;
+    if (processor.type === 'file_download') {
```

Comment:
i wouldn't include this in this endpoint, we have the `/prcoessors/ID/download` endpoint for this